External/FEXCore/Source/Interface/Core/GdbServer.h

```diff
@@ -77,6 +77,7 @@ class GdbServer {
     std::string MemoryMapString{};
     std::string OSDataString{};
 
+    std::array<bool, SignalDelegator::MAX_SIGNALS + 1> PassSignals{};
```

Comment:
Could you document what this member is used for?

Reply:
Added a comment